apps/condo/domains/settings/schema/MobileFeatureConfig.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Generated by `createschema settings.MobileFeatureConfig 'organization:Relationship:Organization:CASCADE; emergencyPhone:Text; commonPhone:Text; onlyProgressionMeterReadingsIsEnabled:Checkbox; meta:Json; ticketSubmittingIsDisabled:Checkbox'`
+ */
+
+const { Text, Relationship, Checkbox } = require('@keystonejs/fields')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/settings/access/MobileFeatureConfig')
+
+
+
+const ERRORS = {
+    TICKET_SUBMITTING_PHONES_NOT_CONFIGURED: {
+        code: BAD_USER_INPUT,
+        type: 'TICKET_SUBMITTING_PHONES_NOT_CONFIGURED',
+        message: 'commonPhone field not specified',
+        messageForUser: 'api.organization.MobileFeatureConfig.TICKET_SUBMITTING_PHONES_NOT_CONFIGURED',
+    },
+    COMMON_PHONE_INVALID: {
+        code: BAD_USER_INPUT,
+        variable: ['data', 'commonPhone'],
+        type: 'COMMON_PHONE_NOT_CONFIGURED',
+        message: 'commonPhone is invalid',
+        messageForUser: 'api.organization.MobileFeatureConfig.COMMON_PHONE_INVALID',
+    },
+    ORGANIZATION_IS_REQUIRED: {
+        code: BAD_USER_INPUT,
+        variable: ['data', 'organization'],
+        type: 'ORGANIZATION_IS_REQUIRED',
+        message: 'The organization is required to create',
+        messageForUser: 'api.organization.MobileFeatureConfig.ORGANIZATION_IS_REQUIRED',
+    },
+}
+
+const MobileFeatureConfig = new GQLListSchema('MobileFeatureConfig', {
+    schemaDoc: 'The entity helps to manage the enabling/disabling of some features in the mobile application,' +
+        ' as well as to store the data required in the disabled state of the feature.',
+    fields: {
+
+        organization: {
+            schemaDoc: 'Ref to the organization. Also needed for scope management',
+            type: Relationship,
+            ref: 'Organization',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        commonPhone: {
+            schemaDoc: 'The phone number where the organization wants to receive common calls',
+            type: Text,
+        },
+
+        ticketSubmittingIsDisabled: {
```

Comment:
Simpler
In natural language: "ticket submission enabled/disabled".

```suggestion
        ticketSubmission: {
```

And don't forget to set default according to business logic.
I suppose, that they should can.

Reply:
Naming the fields like that, I proceeded from a different logic. Now there are 5 fields needed to configure 2 features. When there are 30 fields, finding the right one will be a very difficult task. Therefore, I write the feature prefix and then the field itself. That is, the feature `ticketSubmitting` and then what the field stores. If you change `ticketSubmittingIsEnabled` to `ticketSubmission`, then searching for all fields related to the `ticketSubmitting` feature will be impossible and you will have to manually search for everything.